src/pages/User/components/PullRequests/index.js

```diff
@@ -126,6 +126,30 @@ export default class PullRequests extends Component {
     return count;
   }
 
+  getValidPullRequest(data) {
+    let validPullRequest = [];
+    data.items.forEach((pullRequest, index) => {
+      if (!pullRequest.labels.length) {
+        validPullRequest = [...validPullRequest, pullRequest]
+        return;
+      }
+
+      if (pullRequest.labels.length) {
+        let validFlag = true;
+        pullRequest.labels.forEach((label, index) => {
+          if (label.name === 'invalid') {
+            validFlag = false;
+            return;
+          }
+        });
+        if (validFlag) {
+          validPullRequest = [...validPullRequest, pullRequest]
```

Comment:
Same here. Simplicity. 
```suggestion
          validPullRequest.push(pullRequest);
```